server/src/test/java/io/crate/integrationtests/CopyFromFailFastITest.java

```diff
@@ -152,20 +151,11 @@ public Sessions sqlOperations() {
         }
 
         tmpFileWithLines(rows, "data1.json", target);
-        assertExpectedLogMessages(
-            () -> { // fail_fast = true
```

Comment:
On master we do have log message since it uses `legacyInsert` which emits log entry [here](https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/dml/upsert/TransportShardUpsertAction.java#L237-L246).

Now we use Indexer and exception is swallowed and added to summary [here](https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/engine/indexing/GroupRowsByShard.java#L133-L144).